packages/appkit/src/client/appkit-base-client.ts

```diff
@@ -1715,7 +1715,7 @@ export abstract class AppKitBaseClient {
     const authConnector = ConnectorController.getAuthConnector(namespace)
     const accountState = ChainController.getAccountData(namespace)
     const activeChain = ChainController.state.activeChain as ChainNamespace
-    const activeConnectorId = StorageUtil.getConnectedConnectorId(namespace)
+    const activeConnectorId = StorageUtil.getConnectedConnectorId(namespace || activeChain)
```

Comment:
Maybe we should even remove the storage fetch here? storage should not be used besides initialization in general

A bit scared it will break something tho tbch